components/tidb_query_expr/src/impl_time.rs

```diff
@@ -700,6 +700,33 @@ pub fn add_duration_and_string(
     Ok(Some(res))
 }
 
+#[rpn_fn(capture = [ctx])]
+#[inline]
+pub fn sub_duration_and_string(
+    ctx: &mut EvalContext,
+    arg1: &Duration,
+    arg2: BytesRef,
+) -> Result<Option<Duration>> {
+    let arg2 = std::str::from_utf8(&arg2).map_err(Error::Encoding)?;
+    let arg2 = match Duration::parse(ctx, arg2, MAX_FSP) {
+        Ok(arg) => arg,
+        Err(_) => return Ok(None),
+    };
+
+    let res = match arg1.checked_sub(arg2) {
+        Some(res) => res,
+        None => {
+            return ctx
+                .handle_invalid_time_error(Error::overflow(
+                    "DURATION",
+                    format!("({} + {})", arg1, arg2),
```

Comment:
```suggestion
                    format!("({} - {})", arg1, arg2),
```

Reply:
I changed the + to -
I added the git sign, but DCO is still failing, Is there something else I should add

Reply:
The unit test on the file i changed are passing but when i run the make test command, the make file exists with error code 101, running test on the commit before mine is also giving the same error.
So i'm not sure why it's failing